pkg/app/piped/cloudprovider/ecs/client.go

```diff
@@ -163,12 +160,12 @@ func (c *client) DeleteTaskSet(ctx context.Context, service types.Service, taskS
 	return output.TaskSet, nil
 }
 
-func (c *client) ServiceExists(ctx context.Context, clusterName string, services []string) (bool, error) {
+func (c *client) ServiceExists(ctx context.Context, clusterName string, serviceName string) (bool, error) {
 	input := &ecs.DescribeServicesInput{
 		Cluster:  aws.String(clusterName),
-		Services: services,
+		Services: []string{serviceName},
```

Comment:
should wrap using `aws.StringSlide` since the required type here is `[]*string` not `[]string` 👀

Reply:
I think the required type is `[]string`. ref: https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ecs@v1.1.1#DescribeServicesInput

Reply:
that was version `1.1.1` the latest is `1.38.2` 😂

Reply:
`DescribeServicesInput` you show is in `aws-sdk-go`. I use `DescribeServicesInput` in `awd-sdk-go-v2` like the following.

```rust
type DescribeServicesInput struct {

	// A list of services to describe. You may specify up to 10 services to describe in
	// a single operation.
	//
	// This member is required.
	Services []string

	// The short name or full Amazon Resource Name (ARN)the cluster that hosts the
	// service to describe. If you do not specify a cluster, the default cluster is
	// assumed. This parameter is required if the service or services you are
	// describing were launched in any cluster other than the default cluster.
	Cluster *string

	// Specifies whether you want to see the resource tags for the service. If TAGS is
	// specified, the tags are included in the response. If this field is omitted, tags
	// are not included in the response.
	Include []types.ServiceField
}
```

Reply:
oops, that was my wrong 👍  thank you 🙏